src/main/java/org/reaktivity/nukleus/http_cache/internal/HttpCacheConfiguration.java

```diff
@@ -41,6 +42,8 @@
         HTTP_CACHE_MAXIMUM_REQUESTS = config.property("maximum.requests", 64 * 1024);
         HTTP_CACHE_ETAG_PREFIX = config.property("etag.prefix", new Random().nextInt(99999));
         HTTP_CACHE_ALLOWED_CACHE_PERCENTAGE = config.property("allowed.cache.percentage", 95);
+        HTTP_CACHE_ALLOWED_CACHE_EVICTION_COUNT = config.property("allowed.cache.eviction.count",
```

Comment:
Suggest renaming to `nukleus.http_cache.maximum.cache.eviction.count` as it enforces a maximum limit.